Library/Homebrew/formula.rb

```diff
@@ -922,6 +897,33 @@ def plist_startup
     self.class.plist_startup
   end
 
+  # This method can be used to provide a plist.
+  # @see https://www.unix.com/man-page/all/5/plist/ Apple's plist(5) man page
+  # <pre>plist do
+  #  {
+  #    Label: "#{plist_name}.override",
+  #    ProgramArguments: ["#{opt_bin}/example", "--do-this"],
+  #    RunAtLoad: true,
+  #    KeepAlive: true,
+  #    StandardErrorPath: "/dev/null",
+  #    StandardOutPath: "/dev/null",
+  #  }
+  # end</pre>
+  def plist
+    plists = self.class.plist.map do |block|
+      plist = instance_eval(&block)
+      label = plist[:Label] ||= plist_name
+      [label, plist.to_plist]
+    end
+    plists.empty? ? nil : plists.to_h
+  end
+
+  # @private
+  def self.plist(&block)
```

Comment:
Maybe `plist_hashes` would be a better name here.

Reply:
That breaks everything… unless we then have _another_ method, just to be able to rename this method… is that desirable?
~~~ rb
# @private
def plist(&block)
  self.plist_hashes(&block)
end
~~~

Reply:
As discussed below there's already API breakage so this may make sense to rename as part of that change.